baselines/README.md

```diff
@@ -0,0 +1,47 @@
+# Baseline Testing
+
+This folder contains examples and results for performing testing of baseline controllers for different testcases. 
+
+## Baseline Controller and Test Case Implementation
+
+- The `root/examples/python/controllers/baseline.py` module file is utilized for executing the baseline control for each test case. It simplifies the process by setting `u={}`.
+
+- The test script `run_baselines.py` facilitates the execution of a sample baseline test, with a configuration file `config.json` and the test case name provided as a command-line argument.
+
+## Configuration File Description
+
+- The `config.json` file is used to configure settings for running baseline simulations. It controls options for defining test scenarios, enabling user-defined tests, and saving results. 
+- The `config.json` file includes predefined representative scenarios for all the testcases with different combinations of electricity pricing and time periods. Two-week simulation are conducted with three electricity price schemes and representative time periods for each testcase. For each predefined representative scenario, one week simulation is conducted for the warm-up and two-week simulation (one week before `time_period` and one week after) is then conducted. Please see `root/Testcases/README.md` for different predefined scenarios for each testcase.
+- **`save_kpi_results`**: Save calculated key performance indicator (KPI) results (`true` or `false`).
+- **`save_measurements`**: Save measurement outputs (`true` or `false`).
+- **`run_user_defined_test`**: Enable user-defined scenarios (`true` or `false`).
+- **`user_defined_test_options`**: Custom parameters for user-defined scenarios:
+  - `electricity_price`: Type of electricity pricing.
+  - `start_time`: Start time of the simulation.
+  - `warmup_period`: Warm-up period duration.
+  - `length`: Duration of the simulation.
+
+
+## Run an Example Baseline Test
+
+- Run tests for a specific test case with the command: `$ make run_baseline_<TestCase>`. 
+- Run all the baseline testcases with command: `$ make run_baseline_all`.
+
+## Baseline Testing Results
```

Comment:
Specify somewhere the version of BOPTEST that was used to generated the results that are stored in the csv files currently as committed in the repo. Preferably, these results are from a tagged version (e.g. 0.5.0 or 0.6.0).  This will need to be updated periodically as results are updated with new BOPTEST versions.

Reply:
Version specified "## Baseline Testing Results (for BOPTEST v0.6.0)"